webcompat/static/js/lib/bugform.js

```diff
@@ -155,6 +156,18 @@ function BugForm() {
     img.src = dataURI;
   };
 
+  // Event handler for image file drop; purpose to detect multiple drops
+  this.handleImageDrop = function() {
+    var imageFilled = false;
+    if ($(".wc-UploadForm-label.is-hidden").length) {
+      imageFilled = true;
+    }
+    if (imageFilled) {
+      // if 2nd drag and drop make wrapper hidden to make remove button available
+      $(".wc-UploadForm-wrapper").addClass("is-hidden");
```

Comment:
Hm, I'll have to think this over. It feels a bit too accidental to hide a different DOM element to get this to work. I still think we should be listening for `drop` or `dropend` events and handling things from there, if we want to do this in a more robust way.

But maybe I'm missing something. Will think more about this.